### ISSUE TITLE
helm 2.14 in docker image 2.13.1

### ISSUE THREAD

**Issue by gorilych** [dtzar/helm-kubectl#32]
Travis pushed helm 2.14 to docker image with tag 2.13.1 because of commit 41c8068b33d229938bfa3bb5ce5f4b0f6532f630. This breaks my CI :(

**Comment by gorilych** [comment]
```
$ sudo docker run -ti dtzar/helm-kubectl:2.13.1 bash
Unable to find image 'dtzar/helm-kubectl:2.13.1' locally
2.13.1: Pulling from dtzar/helm-kubectl
e7c96db7181b: Already exists
940217d06d3e: Pull complete
6b209279bd01: Pull complete
Digest: sha256:0910788ddf4c3e0953c1e8f762a0484362db40e4095eab6afcf5555df7334161
Status: Downloaded newer image for dtzar/helm-kubectl:2.13.1
bash-4.4# helm version
Client: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
Error: Get http://localhost:8080/api/v1/namespaces/kube-system/pods?labelSelector=app%3Dhelm%2Cname%3Dtiller: dial tcp 127.0.0.1:8080: connect: connection refused
```

**Comment by gorilych** [comment]
Previous 2.13.1 image is `dtzar/helm-kubectl@sha256:67571a132548d8302b9f984ffa64a7f0858eca67366f7f9e595bcaa93158408e`

**Comment by abuisine** [comment]
I get the exact same problem, a workaround is to use the v2.13.0 which is not impacted

**Comment by dtzar** [comment]
Sorry, just noticed this!  I'll fix the 2.13.1 image in a bit.

**Comment by dtzar** [comment]
This is all fixed up now.  I had to renew the Dockerhub to GitHub connection for automated builds.
